handlers/new_session.go

```diff
@@ -9,14 +9,19 @@ import (
 )
 
 func NewSession(rw http.ResponseWriter, req *http.Request) {
-	if !services.IsHuman(req) {
+	req.ParseForm()
+	challenge := req.Form.Get("g-recaptcha-response")
+	if !services.IsHuman(challenge) {
```

Comment:
Why did you make this change? `IsHuman` tells nothing about recaptcha, so it makes a lot more sense to send the whole request.